docs/user-api/backend-api/resources/tracking/zone/index.md

```diff
@@ -687,3 +697,71 @@ ignored.
 
 From `Placemark` with `MultiGeometry` geometry will be created several geofences.
 If `Placemark.name` defined it will be used as geofence name with respect of hierarchy of `Folder` and `Document`.
+
+
+### `download`
+
+Download geofences as KML File.
+
+#### Parameters
+
+| name     | description                                                                           | type      |
+|:---------|:--------------------------------------------------------------------------------------|:----------|
+| format   | Optional. File format, either "kml" or "kmz". Default is "kml".                       | string    |
+| zone_ids | Optional. Array of geofence IDs. If null, all available geofences will be downloaded. | int array |
+
+#### Example
+
+=== "cURL"
+
+    ```shell
+    curl -X POST '{{ extra.api_example_url }}/zone/download' \
+        -H 'Content-Type: application/json' \
+        -d '{"hash": "22eac1c27af4be7b9d04da2ce1af111b"}'
+    ```
+
+#### Response
```

Comment:
A KML/KMZ file with the points (standard file download).

!!! example "Example of a KML file with different geofences"
...